doc/dev/rbd-diff.rst

```diff
@@ -67,3 +67,80 @@ End
 ---
 
 - u8: 'e'
+
```

Comment:
I would like to make every tag as simple as possible. Then I think we can introduce a new tags for snap_timestamp and CRC in diffs. This way, we can also keep the backward compatibility, we only need to set the timestamp when we found a tag for it.